docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,402 @@
+:toc: macro
+
+= Run Random Beacon
```

Comment:
This should really be “Running a keep client”, right?

Reply:
Maybe?  `keep-client` is kind of ambiguous especially when we have a `keep-client` and `keep-ecdsa`.  The client is effectively the beacon, so I called it that.  Happy to change it if that's a weird line of thought.

Reply:
In my mind, the beacon is the collection of clients acting in concert, and if there's a single entity that we can call “the beacon” it's the smart contracts. Maybe “Running the core keep client”? Trying to think of a good term that we don't have to come back and revise if and when we merge keep-ecdsa into it.